pkg/common/encoding/javaType/javaType.go

```diff
@@ -0,0 +1,119 @@
+package javaType
```

Comment:
Please rename `javaType` to `java_type` for both package and directory.

Reply:
done